actuator-firmware/src/uavcan/feedback_publisher.cpp

```diff
@@ -8,27 +8,43 @@
 #include "analog_input.h"
 #include <error/error.h>
 
+/** Read the pressure of given sensor, returns zero on error*/
+static uint32_t read_pressure(int sensor)
+{
+    // TODO(mspieler): Why is a reset cycle required if the addressed sensor changes?
+    palClearPad(GPIOA, GPIOA_PRESSURE_RST);
+    chThdSleepMilliseconds(3);
+    palSetPad(GPIOA, GPIOA_PRESSURE_RST);
+    chThdSleepMilliseconds(3);
+
+    mpr_start_measurement(&pressure_sensors[sensor]);
+    chThdSleepMilliseconds(5); // Datasheet says wait at least 5m for conversion  (Datasheet p.15)
+
+    // error condition
+    const uint8_t status = mpr_read_status(&pressure_sensors[sensor]);
+    if (mpr_status_is_error(status) || mpr_status_is_busy(status)) {
+        WARNING("Pressure sensor %d status error: 0x%x", sensor, status);
```

Comment:
If that happens, can we somehow reflect this in the UAVCAN node status?

Reply:
done